TreasureHunt/Assets/2.Scripts/MakerSceneManager.cs

```diff
@@ -28,8 +28,13 @@ public void uploadToCloud()
 		camera.targetTexture = rt;
 		camera.Render();
 		RenderTexture.active = rt;
-		Rect temp = new Rect (0, (height - width)/2, width, width);
-		tex.ReadPixels(temp, 0, 0);
+
+		/* hyunSo
+		 * variable cropImageRect for cropping image boundary
+		 * for reducing image file size and decrease useless recognition features
+		 */
```

Comment:
Nice that you added the comment. Here's a suggestion to place instead:

`// Note that Vuforia has a 2MB limit on the image to be uploaded. The image is cropped here to ensure we don't exceed that limit.`

That being said, is there a way to know and restrict the size (within the 2MB limit) before attempting to upload? I worry that with an extremely big image cropping this way won't be enough to prevent the problem from occurring.